script/run-comp-match

```diff
@@ -195,6 +195,9 @@ def archive_match_recordings(archives_dir: Path) -> None:
         # The textures are always the same, so we don't really care if they're
         # already there.
         pass
+    except FileNotFoundError:
```

Comment:
I feel like this would be better as a `if (recording_stem.parent / 'textures').is_dir():`

Reply:
changed in 8c52a34dc89022939efbaa957a0f75d2e7a8a47e